src/storybook/components/related-components/descriptions/alert-banner-description.jsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from "react";
+import { RelatedComponent } from "../../related-component/related-component";
+import AlertBanner from "../../../../components/AlertBanner/AlertBanner";
+import AlertBannerText from "../../../../components/AlertBanner/AlertBannerText/AlertBannerText";
+
+export const AlertBannerDescription = () => {
+  const style = {
```

Comment:
please always move the creation of the style object to be inside the useMemo function.
otherwise it will be create again for each render of the component.